### PR TITLE
Support for urls with repo variables (bnc#944505)

### PR DESCRIPTION
**Pull request by ancorgs**
Needs https://github.com/yast/yast-pkg-bindings/pull/50

### CODE REVIEW

File: src/clients/repositories.rb [hunk 1]
```diff
@@ -323,15 +327,20 @@ def repoInfoRichText(name, raw_url, category)
...
       )
 
+      url = _("Unknown") if url == ""
       raw_url = _("Unknown") if raw_url == ""
 
-      url = Builtins.sformat(_("URL: %1"), raw_url)
+      url_string = Builtins.sformat(_("URL: %1"), url)
+      if url != raw_url
+        url_string += "<BR>"
+        url_string += Builtins.sformat(_("Raw URL: %1"), raw_url)
```
Comment (lslezak) [inline src/clients/repositories.rb]: NP: What about `_("Raw URL: %s") % raw_url` to avoid adding an obsolete builtin...?

Comment (ancorgs) [inline src/clients/repositories.rb]: Hmm, I should probably add a comment for translators here explaining the meaning of "Raw URL")

Comment (lslezak) [inline src/clients/repositories.rb]: Yes, that's a good idea.

**Comment by lslezak** [comment]
Except the small translation issue LGTM

**Comment by lslezak** [comment]
LGTM, thanks!

**Comment by lslezak** [comment]
Note: Do not forget to send the new POT file to Karl, see the "Text Freeze, SP1" mail on yast-devel@.

**Comment by ancorgs** [comment]
@lslezak rebased

**Comment by ancorgs** [comment]
No relevant conflicts resolved in the rebase and travis is still green. So merging.

**Comment by lslezak** [comment]
Thanks, the LGTM still applies :smirk:

**Pull request merged**